### ISSUE TITLE
"indent" rule for "switch" statement

### ISSUE THREAD

**Issue by mistadikay** [eslint/eslint#3024]
code:
```js
something.on('get', e => {
    if (e) {
        switch (e.data) {
            case 'tree':
                console.log('tree');
                break;
            default:
                console.log('default');
        }
    }
});
```

.eslintrc:
```yaml
parser: babel-eslint
  plugins:
    - babel
...
indent:
  - 1
  - 4
```

error:
```shell
  4:13  warning  Expected indentation of 8 characters   indent
  5:17  warning  Expected indentation of 12 characters  indent
  6:17  warning  Expected indentation of 12 characters  indent
  7:13  warning  Expected indentation of 8 characters   indent
  8:17  warning  Expected indentation of 12 characters  indent
```

expected result:
* 4, 7 indentation of 12 chars should not cause warning
* 5, 6, 8 indentation of 16 chars should not cause warning

**Comment by gyandeeps** [comment]
Thats happening because you dont have option `{ indentSwitchCase: true }` set.

**Comment by mistadikay** [comment]
@gyandeeps yep, you're right, `indentSwitchCase` option helped. Thanks!
